server/grammar/YmlParser.g4

```diff
@@ -116,27 +134,46 @@ localField:
 ;
 
 returnField:
-    fieldArrow= FIELD_INTRO optionName=RETURN
-    (
-        optionValues+=valueOrCondition (COMMA optionValues+=valueOrCondition)*?
-    )?
+    fieldArrow= FIELD_INTRO optionName=RETURN optionValue=objectReturnAttributeValue
```

Comment:
I can’t tell much on this file, haha, but I can say that the spacing here is slightly weird and inconsistent around your `=`-s.